keyboards/aki27/cocot46plus/cocot46plus.h

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2022 aki27
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "quantum.h"
+
+typedef union {
+    uint32_t raw;
+    struct {
+        uint8_t cpi_idx;
+        uint8_t scrl_div;
+        uint8_t rotation_angle;
+        int8_t scrl_inv;
+        bool scrl_mode;
+        report_mouse_t last_mouse;
+    };
+} cocot_config_t;
+
+
+
+extern cocot_config_t cocot_config;
+
+enum cocot_keycodes {
+
+    COCOT_SAFE_RANGE = SAFE_RANGE,
+    CPI_SW,
+    SCRL_SW,
+    ROT_R15,
+    ROT_L15,
+    SCRL_MO,
+    SCRL_TO,
+    SCRL_IN,
+
+};
+
+#define CPI_SW QK_USER_0
+#define SCRL_SW QK_USER_1
+#define ROT_R15 QK_USER_2
+#define ROT_L15 QK_USER_3
+#define SCRL_MO QK_USER_4
+#define SCRL_TO QK_USER_5
+#define SCRL_IN QK_USER_6
```

Comment:
You should not use user keycodes at the keyboard level

Reply:
Thanks, I'll move these to the keymap level.

Reply:
`QK_KB_0` and such should be used here rather than `QK_USER_0` and such.